vars/tmsUpload.groovy

```diff
@@ -37,7 +37,11 @@ import static com.sap.piper.Prerequisites.checkScript
     /**
      * Can be used as the description of a transport request. Will overwrite the default. (Default: Corresponding Git Commit-ID)
      */
-    'customDescription'
+    'customDescription',
```

Comment:
[nit-pick] fix is not related to the scope of this PR. Maybe it would be a good idea to use another PR for that. In case there is a battle preventing this PR from being merged fast, the fix for the `customDescription` could merged nevertheless (... faster).